php/src/Google/Protobuf/Internal/DescriptorProto.php

```diff
@@ -307,25 +330,24 @@ public function hasReservedRange()
     }
 
     /**
-     * <pre>
      * Reserved field names, which may not be used by fields in the same message.
      * A given name may only be reserved once.
-     * </pre>
      *
-     * <code>repeated string reserved_name = 10;</code>
+     * Generated from protobuf field <code>repeated string reserved_name = 10;</code>
+     * @return \Google\Protobuf\Internal\RepeatedField
      */
     public function getReservedName()
     {
         return $this->reserved_name;
     }
 
     /**
-     * <pre>
      * Reserved field names, which may not be used by fields in the same message.
      * A given name may only be reserved once.
-     * </pre>
      *
-     * <code>repeated string reserved_name = 10;</code>
+     * Generated from protobuf field <code>repeated string reserved_name = 10;</code>
+     * @param array|\Google\Protobuf\Internal\RepeatedField $var
```

Comment:
Github's [experiencing some delays](https://twitter.com/githubstatus/status/879397164673368066), but this is added in 2f1c48ad0acb677dee9d7853de9e6b53406125c6